src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,6 +5,16 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int chassis;
+    private int enginePower;
+    private int walkingSpeed;
```

Comment:
Don't forget about access modifiers for your custom fields in `Bulldozer` class. You should use private access modifier to encapsulate the fields and provide getters and setters for accessing them.